build/automation/var/profile/dev.mk

```diff
@@ -12,10 +12,5 @@ DOS_API_GATEWAY_PASSWORD := $(or $(DOS_API_GATEWAY_PASSWORD), "")
 DOS_API_GATEWAY_REQUEST_TIMEOUT := 30
 DOS_API_GATEWAY_URL := $(or $(DOS_API_GATEWAY_URL), "//")
 
-TF_VAR_api_gateway_api_key_name := $(PROJECT_ID)-$(PROFILE)-api-key
```

Comment:
Can all these variables be re-added please

Reply:
Resolved